charts/openmetadata/templates/_helpers.tpl

```diff
@@ -35,11 +35,17 @@ Common labels
 */}}
 {{- define "OpenMetadata.labels" -}}
 helm.sh/chart: {{ include "OpenMetadata.chart" . }}
+app.kubernetes.io/component: "{{ .Chart.Name }}"
 {{ include "OpenMetadata.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- if .Values.labels }}
+{{- range $key, $val := fromYaml .Values.labels }}
```

Comment:
remove the `fromYaml` from this condition. Its failing when I tested locally due to that!

Reply:
removed the fromYaml from this condition